catalog.html

```diff
@@ -0,0 +1,217 @@
+                <option value="cheap">Сначала дешёвое</option>
+                <option value="sale">По размеру скидки</option>
+            </select>
+            <a href="#" class="sort-button">
+                <span class="visually-hidden">Показать крупные карточки</span>
+            </a>
+            <a href="#" class="sort-button">
+                <span class="visually-hidden">Показать карточки</span>
+            </a>
+            <a href="#" class="sort-button" >
+                <span class="visually-hidden">Показать список</span>
+            </a>
+
+            <ul class="product-list">
+                <li class="product-card">
+                  <a class="product-card-link" href="#"><!--ссылка на галерею этого отеля-->
+                    <img class="product-card-image" src="#" alt="Amara Resort & Spa">
+                  </a>
+                  <a href="#"><!--ссылка на описание отеля, та же, что будет в "подробнее". тут ведь можно класс не ставить, верно?-->
+                    <h3 class="product-card-title">Amara Resort & Spa</h3>
+                  </a>
+                    <p class="product-card-housing">Гостиница</p>
+                    <p class="product-card-price">от<span>4000</span>₽</p><!--обернула в спан т.к. это значение динамическое, инфляция'c-->
+                    <a class="product-card-button-more" href="#">Подробнее</a>
+                    <button type="button" name="to-favorites">В избранное</button><!--я понимаю что переключении в css
+                    можно сделать зеленый цвет кнопке. но как "В изранное" поменять на "В изранном"? Эту смену текста надо как-то заложить
+                  в разметке?-->
+                    <p class="product-card-rating">Рейтинг:<span>8,5</span></p><!--обернула в спан т.к. это значение динамическое-->
+                    <div></div><!--звездочки-->
+                </li>
+                <li class="product-card">
+                    <a class="product-card-link" href="#">
+                      <img class="product-card-image" src="#" alt="Villas at Poco Diablo">
+                    </a>
+                    <a href="#">
+                      <h3 class="product-card-title">Villas at Poco Diablo</h3>
+                    </a>
+                    <p class="product-card-housing">Гостиница</p>
+                    <p class="product-card-price">от<span>5000</span>₽</p><!--кстати, пробелы надо учесть, которые должны отделять число?-->
+                    <a class="product-card-button-more" href="#">Подробнее</a>
+                    <button type="button" name="to-favorites">В избранном</button>
+                    <p class="product-card-rating" href="#">Рейтинг:<span>9,2</span></p>
+                    <div></div><!--звездочки-->
+                </li>
+                <li class="product-card">
+                    <a class="product-card-link" href="#">
+                      <img class="product-card-image" src="#" alt="Desert Quail Inn">
```

Comment:
```suggestion
                    <a class="product-card-link" href="#">
                      <img class="product-card-image" src="#" alt="Desert Quail Inn">
                    </a>
```
Потерян закрывающий тег `</a>`.